theme/cerveza/src/components/_Input.scss

```diff
@@ -363,3 +363,106 @@
     }
 }
 
+// ==============================================
+// Switch
+// ==============================================
+
+$switchSize: size(xx-large);
+
+.brew-Switch {
+    display: inline-flex;
+    height: $switchSize;
+    position: relative;
+    cursor: pointer;
+
+    div {
```

Comment:
We wouldn't want to have CSS that just references an HTML tag.  Everything should be class based so that the HTML structure can change while the styles stay the same.